### PR TITLE
Combat Knife Sprite Change

### PR DESCRIPTION
**Pull request by Steelpoint**
The Combat Knife now has a newish sprite
knife

Thank you.

:cl: Steelpoint
tweak: Combat knife sprite has been altered slightly. 
/:cl:

### CODE REVIEW

**Comment by LanCartwright** [comment]
i want to kill less people not more steelpoint

come on man

**Comment by Armhulen** [comment]
f

File: code/game/objects/items/weapons/kitchen.dm [hunk 1]
```diff
@@ -101,7 +101,7 @@
 
 /obj/item/weapon/kitchen/knife/combat
 	name = "combat knife"
-	icon_state = "buckknife"
+	icon_state = "combatknife"
```
Comment (ChangelingRain) [inline code/game/objects/items/weapons/kitchen.dm]: a t o m i z e

**Review by ChangelingRain — CHANGES_REQUESTED** [review]

**Comment by optimumtact** [comment]
the sky is blue

**Comment by Xhuis** [comment]
I think this kind of looks ugly compared to the old one. A lot more militaristic, which I don't think fits the miners who get it; they're not military, especially because the only weapons they get are engineered to work in vacuum.

**Comment by dannno** [comment]
I like the old one better

**Comment by Steelpoint** [comment]
The old one looks out of place and clunky imo, hence my proposal.

**Comment by Steelpoint** [comment]
Also what is everyone on about?

This is just the miner knife except with a different coloured handle.

**Comment by ChangelingRain** [comment]
Yeah the old one is nicer.

**Pull request closed without merging**